napari/settings/_plugins.py

```diff
@@ -25,6 +25,16 @@ class PluginAPI(str, Enum):
 
 
 class PluginsSettings(EventedModel):
+    use_npe2_adaptor: bool = Field(
+        False,
+        title=trans._("Use npe2 adaptor"),
+        description=trans._(
```

Comment:
maybe out-of-scope, but it would be cool to reflect this description into a tooltip for the corresponding thing in the properties panel.

Reply:
that was actually the original intention for these descriptions.  but it looks like preferences are no longer doing this.  yeah: another PR